src/Workspaces/Core/Portable/Microsoft.CodeAnalysis.Workspaces.csproj

```diff
@@ -323,6 +323,7 @@
     <InternalsVisibleTo Include="Microsoft.CodeAnalysis.Razor.Workspaces" Key="$(RazorKey)" />
     <InternalsVisibleTo Include="Microsoft.VisualStudio.Editor.Razor" Key="$(RazorKey)" />
     <InternalsVisibleTo Include="Microsoft.VisualStudio.LanguageServices.Razor" Key="$(RazorKey)" />
+    <InternalsVisibleTo Include="FSharp.Editor" Key="$(FSharpKey)" />
```

Comment:
new interface (IClassificationService) that replaces obsolete (IEditorClassificationService) is defined in Workspace layer

Reply:
this seems unfortunate.  it removes one accessible interface, but replaces it with the entire internal surface area of 'Workspaces'

Reply:
tagging @jasonmalinowski if this is the right wya to do things here.

Reply:
➡️ This is now avoided